packages/website/src/components/Playground.tsx

```diff
@@ -85,6 +85,15 @@ function Playground(): JSX.Element {
     [setState],
   );
 
+  const onLoaded = useCallback(
+    (ruleNames: RuleDetails[], tsVersions: readonly string[]): void => {
+      setRuleNames(ruleNames);
+      setTSVersion(tsVersions);
+      setIsLoading(false);
+    },
+    [],
+  );
+
```

Comment:
This small refactoring was necessary to fix an issue raised by updating one of the dependency arrays on [useSandboxServices.ts](https://github.com/amorimr/typescript-eslint/pull/1/files#diff-4ef6f5cef1c213bd334eb1bf373edec1eae7a1a29f974f437d8f48b6a3c21643):

Since the original `onLoaded` was inlined, it was being recreated at every render, causing racing conditions on `useSandboxServices` effects that crashed the page.